infoschema/infoschema.go

```diff
@@ -354,8 +354,8 @@ func init() {
 		tableInfo := buildTableMeta(name, cols)
 		infoSchemaTables = append(infoSchemaTables, tableInfo)
 		tableInfo.ID = autoid.GenLocalSchemaID()
-		for _, c := range tableInfo.Columns {
-			c.ID = autoid.GenLocalSchemaID()
+		for i, c := range tableInfo.Columns {
```

Comment:
Why make this change?

Reply:
Normally, the column ID started at 1, no need `GenLocalSchemaID ` any more, and I have remove this function.